### ISSUE TITLE
The request to report bugs from within spacemacs references an obsolete keybinding

### ISSUE THREAD

**Issue by coventry** [syl20bnr/spacemacs#5155]
Viz.,

```
If you are posting about a *bug* please close this issue and open a new one
using [SPC h I] or [M-m h I] from within Spacemacs. You can delete this message
to start typing.
```

In my hands, SPC h is bound to helm-resume.  I don't know what function it might be asking users to invoke.  Perhaps this message should be updated to include "(M-x <something>)" in case the binding changes again.

**Comment by NJBS** [comment]
It's asking you to invoke `report-issue` under `SPC h I` (capital i, not l),`helm-resume` lives on `SPC h l` which is what you probably ran into.

**Comment by coventry** [comment]
OK.  On my computer (stock ubuntu fonts), lower case l is practically indistinguishable from upper case I.  Here they are next to each other:

![image](https://cloud.githubusercontent.com/assets/70152/13194313/fde34e7a-d75a-11e5-91e3-1d0919ec8b17.png)

**Comment by coventry** [comment]
Also, I don't have report-issue (emacs-version 25.1.50.2), and there is no SPC h I binding.

**Comment by xfq** [comment]
BTW, the documentation is not clear:

```
| Key Binding     | Description                                                                          |
|-----------------+--------------------------------------------------------------------------------------|
| ~SPC h I~       | Open a page for creating a new Spacemacs issue on GitHub with information pre-filled |
| ~SPC u SPC h I~ | Open a page for creating a new Spacemacs issue on GitHub with information pre-filled |
```

&

```
(defun spacemacs/report-issue (arg)
  "Browse the page for creating a new Spacemacs issue on GitHub,
with the message pre-filled with template and information."
```

They don't mention the differences between these two key bindings.  IIUC the second key binding will include the output of `(view-lossage)`.

Maybe I should report this as a new issue.

**Comment by syl20bnr** [comment]
> OK. On my computer (stock ubuntu fonts), lower case l is practically indistinguishable from upper case I. 

I noticed that when I added the template. I'll update the template to be more explicit.

Labelling this issue `documentation` issue for the problems raised by @xfq

**Comment by nixmaniack** [comment]
This has been clarified in the new issue template. 

> [SPC h I] or [M-m h I] (I = capital i) from within...

Can be closed @StreakyCobra

**Comment by StreakyCobra** [comment]
I think there is still the documentation issue mentionned by xfq

**Comment by nixmaniack** [comment]
@StreakyCobra I see. Opened a PR to fix it.

**Comment by xged** [comment]
Letters l I are indistinguishable on Github normal font (not on my computer though). I was confused as well, even with a text *(I = capital i)* in place. What a weird cause of mistake .. :-)

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
